### PR TITLE
Some_blog: add views 

### PR DESCRIPTION
**Pull request by ambrose-kibet**
## In this milestone we:
- [ ] Add view to our blog 
- [ ]  update request tests

### CODE REVIEW

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @ambrose-kibet 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

- [ ] Please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="img-container">
+    <img src="<%= user.photo %>" alt="<%= user.name %> photo" width="100%" height="100%">
+  </div>
+  <div class="user-card px-2">
+    <h3 class="text-start">
+      <%= link_to user.name, user_path(user) %>
+    </h3>
+    <p class="text-end">Number of posts: <%= user.posts_counter %></p>
+  </div>
+</article>
```
Comment (Gambit142) [inline app/views/users/_user.html.erb:11]: - [ ] Great use of partials to render page sections that are the same across different pages. 👏

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,12 @@
-<h1>user show placeholder</h1>
\ No newline at end of file
+
+<div class="container">
+<%= render partial: 'user', locals: { user: @user } %>
+<h4 class="text-center">Bio</h4>
+<p class="text-center"><%=@user.bio%></p>
+<% @user.recent_posts.each do |post|%>
+ <%= render partial: 'posts/post', locals: { post: post } %>
```
Comment (Gambit142) [inline app/views/users/show.html.erb:7]: - [ ] The `recent_posts` method should return the three most recent posts of a certain user. Currently, you are returning the first three posts of the user. I would advise you to change the logic for the recent_posts in your `app/models/user.rb` file. 👍 😎 

#### Hint: 
```ruby
def recent_posts
  posts.order(created_at: :desc).limit(3)
end
```

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @ambrose-kibet  👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Created _**index**_ and _**show**_ pages for `User` and `Post` models ✔️ 
- Pages created are identical to the wireframe provided ✔️ 
- Great use of partials ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by TadesseDev — APPROVED** [review]
### Hi @ambrose-kibet 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**